components/camel-jsonpath/src/main/docs/jsonpath-language.adoc

```diff
@@ -31,9 +31,9 @@ The JSonPath language supports 7 options which are listed below.
 | Name | Default | Java Type | Description
 | resultType |  | String | Sets the class name of the result type (type from output)
 | suppressExceptions | false | Boolean | Whether to suppress exceptions such as PathNotFoundException.
-| allowSimple | true | Boolean | Whether to allow in inlined simple exceptions in the json path expression
+| allowSimple | true | Boolean | Whether to allow in inlined simple exceptions in the JSON path expression
 | allowEasyPredicate | true | Boolean | Whether to allow using the easy predicate parser to pre-parse predicates.
-| writeAsString | false | Boolean | Whether to write the output of each row/element as a JSon String value instead of a Map/POJO value.
+| writeAsString | false | Boolean | Whether to write the output of each row/element as a JSON String value instead of a Map/POJO value.
```

Comment:
This documentation come from the Javadoc so you have to edit the Java Code comments and then rebuild the module